packages/docs-ui/README.md

```diff
@@ -54,5 +55,22 @@ import { MenuButton } from '@braid-design-system/docs-ui';
 | open    | `boolean`    | The Menu can either be open or closed. If open, the button will change to a close icon (defaults to `false`). |
 | onClick | `() => void` | A callback function to handle button presses (defaults to `false`).                                           |
 
+### `SideNavigationGroup`
+
+A group of SideNavigationItems, with an optional title.
```

Comment:
🚲🏚️🚨 Bike-shed alert!

Wondering if `SideNavigationList` is a better name, given it is a semantic `nav` element with a list of links. Thoughts? Then the description can talk more about the pattern than the API.

```suggestion
A list of navigation links designed for the side bar.
```

Reply:
I like it, but does it seem like list indicates it's the entire sidebar list, not just a subsection? Maybe something like sublist or section?